test/job/fetch-updates/fetch-updates.helper.test.ts

```diff
@@ -0,0 +1,194 @@
+import helper from '../../../src/job/fetch-updates/fetch-updates.helper';
+
+import { IMergeRequestModel } from '../../../src/api/mongo';
+import { IGitlabMergeRequestDetail, IGitlabMergeRequestReaction, IGitlabUser, IGitlabMergeRequestDiscussion } from '../../../src/api/gitlab';
+
+const getUser = (name: string): IGitlabUser => {
+    return {
+        'username': name,
+    } as IGitlabUser;
+};
+
+const getDiscussion = (user: IGitlabUser, isOpen: boolean) => {
+    return {
+        'individual_note': false,
+        'notes': [{
+            'author': user,
+            'resolvable': true,
+            'resolved': !isOpen,
+        }],
+    } as IGitlabMergeRequestDiscussion;
+};
+
+describe('fetch-updates.job', () => {
+    let currentMR: IMergeRequestModel;
+    let remoteMR: IGitlabMergeRequestDetail;
+
+    beforeEach(() => {
+        currentMR = {
+            'analytics': {
+                'upvoters': [],
+                'reviewers': [],
+            },
+            'slack': {
+                'reactions': [],
+            },
+        } as IMergeRequestModel;
+
+        remoteMR = {
+            'upvotes': 0,
+        } as IGitlabMergeRequestDetail;
+    });
+
+    describe('getUpvoteReactions', () => {
+        let remoteReactions: IGitlabMergeRequestReaction[] = [];
+
+        beforeEach(() => {
+            remoteReactions = [];
+        });
+
+        test('should get zero reactions', () => {
```

Comment:
Poderia  dizer também pq o teste deve ter esse resultado final. Serve para todos os testes. `should ... when ...` ou because.